### ISSUE TITLE
[HOLD for payment 2023-06-02] [$1000] Inconsistent spacing between amount and tick mark

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#19049]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Send a money request from account A to account B
2. Settle the requested amount from account B
3. Look at the "settled up [amount]" and tick mark

## Expected Result:
There should be a spacing between the amount and the tick mark
## Actual Result:
There is no space between the amount and the tick mark

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.14-10
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation
![image](https://github.com/Expensify/App/assets/43996225/80e872d9-dc01-4cd0-b09a-6ff48aa4f6f1)

https://github.com/Expensify/App/assets/43996225/17cd4b13-77f4-4ae0-93a0-f307db896f4f

<img width="960" alt="1" src="https://github.com/Expensify/App/assets/43996225/bb3c03a9-06bc-479f-97a6-1117acfcee2c">

**Expensify/Expensify Issue URL:**
**Issue reported by:** @nathan-mulugeta
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1684143189663259

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0172355caaa1314f42</li>
        <li>Upwork Job ID: 1658893137305018368</li>
        <li>Last Price Increase: 2023-05-17</li>
    </ul>
</details>

**Comment by ahmedGaber93** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
 Inconsistent spacing between amount and tick mark
### What is the root cause of that problem?
no margin between icon and text
### What changes do you think we should make in order to solve the problem?
surround icon with 
```javascript
<View style={styles.iouPreviewBoxCheckmark}>
    <Icon/>
</View>
```
### What alternative solutions did you explore? (Optional)
N/A

**Comment by hungvu193** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.

### What is the root cause of that problem?
We are passing style to the Icon component in here:
https://github.com/Expensify/App/blob/f013fe2d26930bffb90e23620f8360743a0d2ea8/src/components/ReportActionItem/ReportPreview.js#L124
But inside Icon component we don't have `style` props, so we don't see any margin here.
### What changes do you think we should make in order to solve the problem?
In stead of using `style` props we should use `additionalStyles` props for Icon component, we also need to make sure the margin is consistent with the Checkmark in `IOUPreview`.

```javascript
              <Icon
// we should use additionalStyles instead of style here
                     additionalStyles={[styles.ml1]}
                     src={Expensicons.Checkmark}
                     fill={themeColors.iconSuccessFill}
                />
```

### What alternative solutions did you explore? (Optional)

**Comment by lobcare** [comment]
I think if you change code like this, the bug will disappear
App/tree/main/src/styles line 2486:

current :
   iouPreviewBoxCheckmark: {
        marginLeft: 4,
        alignSelf: 'center',
    }
Change it: 
  iouPreviewBoxCheckmark: {
        marginLeft: 5,
        alignSelf: 'center',
    }

Is it easy, right ?

**Comment by ahmedGaber93** [comment]
>Hey @ahmedGaber93 Please make sure to wait until the job is in Upwork before adding a proposal to the GH.

Thanks @Christinadobrzyn.
But guideline [here](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#propose-a-solution-for-the-job) recently changed to allow proposal at any time
>You can propose solutions on any issue at at any time, but if you propose solutions to jobs before the Help Wanted label is applied, you do so at your own risk. Proposals will not be reviewed until the label is added and there is always a chance that we might not add the label or hire an external contributor for the job.

**Comment by jasperhuangg** [comment]
> I think if you change code like this, the bug will disappear App/tree/main/src/styles line 2486:
> 
> current : iouPreviewBoxCheckmark: { marginLeft: 4, alignSelf: 'center', } Change it: iouPreviewBoxCheckmark: { marginLeft: 5, alignSelf: 'center', }
> 
> Is it easy, right ?

Hey @lobcare, please follow the proposal template: https://github.com/Expensify/App/blob/main/contributingGuides/PROPOSAL_TEMPLATE.md

**Comment by Santhosh-Sellavel** [comment]
Thanks @jasperhuangg for the Review! 

@ahmedGaber93 Let us know when is PR is ready for Review!

**Comment by ahmedGaber93** [comment]
@Santhosh-Sellavel [PR](https://github.com/Expensify/App/pull/19222) is ready for review.

**Comment by lobcare** [comment]
I like finding bugs
So I made PR yesterday
But you need my upwork account
I don't have it
In this case, what should I do?
Thanks.

**Comment by Santhosh-Sellavel** [comment]
@lobcare 
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email [contributors@expensify.com](mailto:contributors@expensify.com) to request to join our Slack channel!

Post your questions on the channel, so people will address your queries.
For now, Upwork is mandatory for getting paid.

**Comment by Christinadobrzyn** [comment]
Hired in Upwork - https://www.upwork.com/jobs/~0172355caaa1314f42

- @Nathan-Mulugeta as reporter
- @Santhosh-Sellavel as C+
- @ahmedGaber93 as contributor

@Santhosh-Sellavel what do you think of a regression test for this?

**Comment by Santhosh-Sellavel** [comment]
> @Santhosh-Sellavel what do you think of a regression test for this?

Not needed here as it's a small UI polish cc: @jasperhuangg

**Comment by lobcare** [comment]
Hi, Santhosh
It's me Jack and a new member of Expensify Slack Channel.
Actually I have some problems contributing to Expensify.
I don't have my upwork account now because it is blocked.
I know once it is blocked I can't recover it.
And then I don't know exactly how to work for Expensify as a guest.
If possible, kindly teach me.
If possible, I wanna collaborate with you.
I think I would be helpful to you.
I am waiting for your reply.
Thanks a lot for taking your time.
Best regards.
Jack

**Comment by Santhosh-Sellavel** [comment]
@lobcare I've already replied [here](https://github.com/Expensify/App/issues/19049#issuecomment-1554837933). Further queries post your questions on channel

**Comment by Christinadobrzyn** [comment]
Ready for payment but there are some [additional convos](https://github.com/Expensify/App/pull/19222#issuecomment-1568787522) on the PR - @Santhosh-Sellavel @luacmartins can you confirm if this can be paid or if we're working on something else first?

Do we need a regression test for this?

**Comment by luacmartins** [comment]
@Christinadobrzyn yes, we can pay that one. We're addressing those in a separate issue.

**Comment by Christinadobrzyn** [comment]
Awesome! Thanks for confirming @luacmartins 

Looks like this isn't eligible for a speed bonus (PR created May 18th - merged May 23rd)

Paid in Upwork based on this - https://github.com/Expensify/App/issues/19049#issuecomment-1564949792

Let me know about any regression tests!

**Comment by Santhosh-Sellavel** [comment]
> **BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:
> 
> * [x]  [@Santhosh-Sellavel] The PR that introduced the bug has been identified. Link to the PR: https://github.com/Expensify/App/pull/18486
> * [x]  [@Santhosh-Sellavel] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment: https://github.com/Expensify/App/pull/18486/files#r1215085067
> * [x]  [@Santhosh-Sellavel] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion: N/A Should be caught based on the check list
> * [x]  [@Santhosh-Sellavel] Determine if we should create a regression test for this bug. N/A
> * [x]  [@Santhosh-Sellavel] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again. N/A
> * [x]  [@Christinadobrzyn] Link the GH issue for creating/updating the regression test once above steps have been agreed upon: N/A
